src/Command/MauticLanguagePackerCommand.php

```diff
@@ -0,0 +1,131 @@
+
+            return Command::FAILURE;
+        }
+
+        $packagesDir     = $this->parameterBag->get('mlp.packages.dir');
+        $translationsDir = $this->parameterBag->get('mlp.translations.dir');
+
+        $filterLanguages = $input->getArgument('filter-languages');
+        $uploadPackage   = $input->getOption('upload-package');
+        $language        = $input->getOption('language') ?? '';
+
+        // Remove any previous pulls and rebuild the translations folder
+        $translationsDirEvent = new PrepareDirEvent($translationsDir, true);
+        $this->eventDispatcher->dispatch($translationsDirEvent, PrepareDirEvent::NAME);
+
+        // Fetch the project resources now and store them locally
+        $resourceEvent = new ResourceEvent($io, $filterLanguages, $translationsDir, $language);
+        $this->eventDispatcher->dispatch($resourceEvent, ResourceEvent::NAME);
+
+        // Now we start building our ZIP archives
+        $packagesDirEvent = new PrepareDirEvent($packagesDir);
+        $this->eventDispatcher->dispatch($packagesDirEvent, PrepareDirEvent::NAME);
+
+        // Add a folder for our current build
+        $timestamp            = (new \DateTime())->format('YmdHis');
+        $packagesTimestampDir = $packagesDir.'/'.$timestamp;
+        $packages2DirEvent    = new PrepareDirEvent($packagesTimestampDir);
+        $this->eventDispatcher->dispatch($packages2DirEvent, PrepareDirEvent::NAME);
+
+        // Compile our data to forward to mautic.org and build the ZIP packages
+        $createPackageEvent = new CreatePackageEvent($io, $filterLanguages, $translationsDir, $packagesTimestampDir);
+        $this->eventDispatcher->dispatch($createPackageEvent, CreatePackageEvent::NAME);
+
+        $errorsFile = $translationsDir.'/errors.txt';
+
+        if (file_exists($errorsFile)) {
+            $io->warning(
+                sprintf('Created language packages for Mautic. Check %1$s as there were errors!', $errorsFile)
+            );
+        } else {
+            $io->success('Successfully created language packages for Mautic!');
+        }
+
+        // If instructed, upload the packages
+        if ($uploadPackage) {
+            $io->info('Starting package upload to AWS S3, checking credentials.');
+            if (
+                !$this->parameterBag->get('mlp.aws.key')
+                || !$this->parameterBag->get('mlp.aws.secret')
+                || !$this->parameterBag->get('mlp.aws.region')
+                || !$this->parameterBag->get('mlp.aws.bucket')
+            ) {
+                $io->error('Add AWS_KEY, AWS_SECRET, AWS_REGION and AWS_BUCKET in .env.');
+
+                return Command::FAILURE;
+            }
+
+            $uploadPackageEvent = new UploadPackageEvent(
+                $io,
+                $packagesTimestampDir,
+                $this->parameterBag->get('mlp.aws.bucket')
+            );
+            $this->eventDispatcher->dispatch($uploadPackageEvent, UploadPackageEvent::NAME);
```

Comment:
```suggestion
            $this->eventDispatcher->dispatch($uploadPackageEvent);
```
Let's save some code lines by omitting the optional `$eventName` parameter.